### PR TITLE
[server][meshsync] hard reset database

### LINKED ISSUE #8503: Data persisted in few tables even after resetting database
### Current Behavior
When we reset the database, data in few tables are persisted.
### Expected Behavior

All the tables should be reset.

### Screenshots/Logs

https://github.com/meshery/meshery/assets/64671908/d2d988a5-83c6-4c4f-b66a-0893a631b18f

### Environment

- **Host OS:** Windows
- **Platform:** Docker
- **Meshery Server Version:** stable-v
- **Meshery Client Version:** stable-v

---
### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 📚 [Instructions for contributing to documentation](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by VihasMakwana**
The hard reset worked partially before because we didn't reset Registries. Reset all known tables and create fresh ones with no data after.

**Notes for Reviewers**

This PR fixes #8503

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by MUzairS15 — APPROVED** [review]
Don’t drop events table.

**Review by MUzairS15 — CHANGES_REQUESTED** [review]
Don’t drop events table.

**Comment by VihasMakwana** [comment]
> Don’t drop events table.

understood.

File: server/internal/graphql/resolver/meshsync.go [hunk 1]
```diff
@@ -66,28 +68,22 @@ func (r *Resolver) resyncCluster(ctx context.Context, provider models.Provider,
...
 				r.Log.Error(err)
 				return "", err
 			}
 
+			for _, table := range tables {
+				if table == "events" {
+					continue
+				}
+				if err := dbHandler.Migrator().DropTable(table); err != nil {
+					r.Log.Error(err)
```
Comment (leecalcote) [inline server/internal/graphql/resolver/meshsync.go:82]: @VihasMakwana can this be a MeshKit-style error?

https://docs.meshery.io/project/contributing/contributing-error

**Review by leecalcote — APPROVED** [review]
Good work, @VihasMakwana 🥇

**Pull request merged**